lib/msal-browser/rollup.config.js

```diff
@@ -211,4 +211,96 @@ export default [
             })] : []),
         ],
     },
+    {
+        // Redirect Bridge - ES module build
+        input: "src/redirect_bridge/index.ts",
```

Comment:
You can make this part of the existing config by just making the input param an array - will be faster to build with less duplication in the final package. Like common does here: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/c2ff4da426eed35c8e3aca9feed5fd014fded983/lib/msal-common/rollup.config.js#L17